config/sidekiq-schedule.yml

```diff
@@ -41,6 +41,11 @@ mark_outdated_reputation_year_periods:
   queue: reputation
   args: ["year"]
 
+send_mentor_discussion_nudges:
+  cron: "15 0 * * *"
```

Comment:
@iHiD You wanted this to run every hour, correct?

Reply:
Yes.